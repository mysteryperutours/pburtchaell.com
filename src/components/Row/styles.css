.row {
  position: relative;
  display: flex;
  flex: 0 auto;
  flex-direction: row;
  flex-wrap: wrap;
  box-sizing: border-box;
  align-content: auto;
  width: 100%;
  max-width: 1400px;
  margin: 0 auto;
}
