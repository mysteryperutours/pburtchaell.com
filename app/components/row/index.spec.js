import React from 'react';
import Row from './index';

describe(Row.name, () => {
  it('should render component with default sized row', () => {
    expect(shallow(
      <Row />
    )).toMatchSnapshot();
  });

  it('should render component with large sized row', () => {
    expect(shallow(
      <Row size="large" />
    )).toMatchSnapshot();
  });

  it('should render component with medium sized row', () => {
    expect(shallow(
      <Row size="medium" />
    )).toMatchSnapshot();
  });
});
