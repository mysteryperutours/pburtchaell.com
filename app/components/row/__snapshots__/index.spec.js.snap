// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Row should render component with default sized row 1`] = `
<div
  className="row-container row-container--default"
  size="default"
>
  <div
    className="row"
  />
</div>
`;

exports[`Row should render component with large sized row 1`] = `
<div
  className="row-container row-container--large"
  size="large"
>
  <div
    className="row"
  />
</div>
`;

exports[`Row should render component with medium sized row 1`] = `
<div
  className="row-container row-container--medium"
  size="medium"
>
  <div
    className="row"
  />
</div>
`;
